/**:
  ros__parameters:
    # MPC params
    mode: "spiralMPC_eMPC"
    horizon: 30
    time_step: 0.1
    param_set: "P1"
    solver_opts:
      ipopt.tol: 1e-5
    # Trajectory params
    # traj_shape: "hover"
    # traj_shape: "generate_circle"
    traj_shape: "circle_r_1.2_sPerFullCircle_30"
    # traj_shape: "generate_sin"
    # traj_shape: "load=traj_01.yaml"
    traj_duration: 10
    tuning:
      P1:
        # Q: [1, 1, 1, 1, 0]
        Q: [1, 1, 1, 1, 1]
        R: [0.1, 0.1, 0.01]
        P_mult: 1
        # Attention: The following parameters are only updated if the terminal set is recalculated!
        k_omega: 1.0
        max_acceleration: 0.0    
        time_scaling: 5
        empc_horizon: 10
        # empc_horizon: 10
      P2:
        Q: [1, 1, 1, 1, 1]
        # Q: [5, 5, 0.2, 0.2, 0.1]
        # R: [0.1, 0.1, 0.1]
        R: [0.1, 0.1, 0.01]
        P_mult: 1
        # Attention, see above
        k_omega: 0.1
        max_acceleration: 0.0    
    actuator_failures: 
      # # original
      # - act_ids: [3,1]
      #   intensity: 1
      #   start_time: 0
      # - act_ids: [4,1]
      #   intensity: 1
      #   start_time: 0
    
      # # opposite
      # - act_ids: [3,2]
      #   intensity: 1
      #   start_time: 0
      # - act_ids: [4,2]
      #   intensity: 1
      #   start_time: 0

      # - act_ids: [1,1]
      #   intensity: 1
      #   start_time: 0
      # - act_ids: [2,1]
      #   intensity: 1
      #   start_time: 0

      - act_ids: [4, 1]
        intensity: 1
        start_time: 0
      - act_ids: [2, 2]
        intensity: 1
        start_time: 0

      # - act_ids: [2, 2]
      #   intensity: 1
      #   start_time: 0
      # - act_ids: [3, 2]
      #   intensity: 1
      #   start_time: 0
      # - act_ids: [2, 1]
      #   intensity: 1
      #   start_time: 0

      # - act_ids: [3, 1]
      #   intensity: 1
      #   start_time: 0
      # - act_ids: [4, 1]
      #   intensity: 1
      #   start_time: 0

      # - act_ids: [3, 2]
      #   intensity: 1
      #   start_time: 0
      # - act_ids: [2, 2]
      #   intensity: 1
      #   start_time: 0
      # - act_ids: [3, 1]
      #   intensity: 1
      #   start_time: 0

